packages/iframe-stamper/src/index.ts

```diff
@@ -67,13 +67,7 @@ export class IframeStamper {
       );
     }
 
-    const container = document.getElementById(config.iframeContainerId);
-    if (!container) {
-      throw new Error(
-        `Cannot create iframe stamper: no container with ID ${config.iframeContainerId} exists in the current document`
-      );
-    }
-    this.container = container;
+    this.container = config.iframeContainer;
```

Comment:
Separately: do we need to check for `config.iframeContainer !== undefined`? Or is it already implicit if the type is HTMLElement?

Reply:
Good point! Looked into this and the `document.getElementById` and `document.querySelector` both return `HTMLElement | null` when the element's not found. However, they can return `undefined` when they're accessed before the element is created. In short, yep I added this check. Thanks!